canvas_modules/harness/assets/images/custom-canvases/detached-links/decorations/dragStateArrow.svg

```diff
@@ -0,0 +1,2840 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 24.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.0//EN" "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd" [
+	<!ENTITY ns_extend "http://ns.adobe.com/Extensibility/1.0/">
+	<!ENTITY ns_ai "http://ns.adobe.com/AdobeIllustrator/10.0/">
+	<!ENTITY ns_graphs "http://ns.adobe.com/Graphs/1.0/">
+	<!ENTITY ns_vars "http://ns.adobe.com/Variables/1.0/">
+	<!ENTITY ns_imrep "http://ns.adobe.com/ImageReplacement/1.0/">
+	<!ENTITY ns_sfw "http://ns.adobe.com/SaveForWeb/1.0/">
+	<!ENTITY ns_custom "http://ns.adobe.com/GenericCustomNamespace/1.0/">
+	<!ENTITY ns_adobe_xpath "http://ns.adobe.com/XPath/1.0/">
+]>
+<svg version="1.0" id="Layer_1" xmlns:x="&ns_extend;" xmlns:i="&ns_ai;" xmlns:graph="&ns_graphs;"
+	 xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 20 20"
+	 enable-background="new 0 0 20 20" xml:space="preserve">
+<switch>
+	<foreignObject requiredExtensions="&ns_ai;" x="0" y="0" width="1" height="1">
+		<i:aipgfRef  xlink:href="#adobe_illustrator_pgf">
+		</i:aipgfRef>
+	</foreignObject>
+	<g i:extraneous="self">
+		<g>
+			<circle fill="#FFFFFF" cx="10" cy="10" r="10"/>
+			<path fill="#0F62FE" d="M10,2c4.41,0,8,3.59,8,8s-3.59,8-8,8s-8-3.59-8-8S5.59,2,10,2 M10,0C4.48,0,0,4.48,0,10s4.48,10,10,10
+				s10-4.48,10-10S15.52,0,10,0L10,0z"/>
+		</g>
+		<polygon fill="#0F62FE" points="8.59,16 7,14.51 11.82,10 7,5.49 8.59,4 15,10 		"/>
+	</g>
+</switch>
+<i:aipgf  id="adobe_illustrator_pgf" i:pgfEncoding="zstd/base64" i:pgfVersion="24">
```

Comment:
Are all these required for the svg image? or can it be cleaned up?

Reply:
In Elyra Canvas they are just used in the sample applications so they are not shipped with the product. They do however exist in Modeler Flows product. I'd noticed this before so we already have this issue for that problem: 
https://github.ibm.com/NGP-TWC/wml-canvas-planning/issues/8451 

I also opened one in wdp-abstract-canvas as a reminder to add those images into the sample apps when we get them back from design.
https://github.ibm.com/NGP-TWC/wdp-abstract-canvas/issues/2955